scripts/screeps_rl_training_runner.py

```
@@ -3357,13 +3369,53 @@ def policy_update_learning_rate(policy_gradient: JsonObject) -> float:
+def bounded_policy_parameter_integer_step_update(
+    *,
+    anchor_value: float,
+    gradient_value: float,
+    learning_rate: float,
+    spec: JsonObject,
+) -> float | int:
+    if abs(gradient_value) <= 1e-12:
+        return bounded_policy_parameter_value(anchor_value, spec)
+    step = number_or_none(spec.get("step"))
+    step_float = float(step) if step is not None and float(step) > 0 else 1.0
+    step_count = max(1, int(round(abs(float(learning_rate)))))
```

Ruff (0.15.13)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)